react/components/UploadModal/ReportInformationDetails.tsx

```diff
@@ -23,7 +23,7 @@ const ReportInformationDetails = ({
         })}
       >
         {t(`reportInformationInitial${variant}`, {
-          a: (content: string) => <a href="/">{content}</a>,
+          'file-name-link': (content: string) => <a href="/">{content}</a>,
```

Comment:
When integration done the idea is to use the fileName here and the content as a fallback